create_ibu10m_items.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+# from tqdm import tqdm
+from scripts.ibu10m.item import create_ibu10m_item
+
+SAMPLE_DIR = "/Users/joshua.chung/clms-stac/scripts/ibu10m/IBU_2018_010m_al_03035_v010"
```

Comment:
this is hardcoded

Reply:
Ideally the script should go through all countries, i.e., `IBU_2018_010m_al_03035_v010`, `IBU_2018_010m_ch_03035_v010`, etc.

In our case, we are focusing on items (tiles) and we don't want duplicated items (tiles). I would suggest using glob library to obtain all tiffs in ibu10m and filtering out the duplicated tiles.